tests/test_keras_integration.py

```diff
@@ -250,6 +286,30 @@ def test_push_to_hub(self):
             git_user="ci",
             git_email="ci@dummy.com",
             config={"num": 7, "act": "gelu_fast"},
+            include_optimizer=False,
+        )
+
+        model_info = HfApi(endpoint=ENDPOINT_STAGING).model_info(
+            f"{USER}/{REPO_NAME}",
+        )
+        self.assertEqual(model_info.modelId, f"{USER}/{REPO_NAME}")
+
+        self._api.delete_repo(name=f"{REPO_NAME}", token=self._token)
+
+    def test_push_to_hub_model_kwargs(self):
+        REPO_NAME = repo_name("PUSH_TO_HUB")
+        model = self.model_init()
+        model.build((None, 2))
+        push_to_hub_keras(
+            model,
+            repo_path_or_name=f"{WORKING_REPO_DIR}/{REPO_NAME}",
+            api_endpoint=ENDPOINT_STAGING,
+            use_auth_token=self._token,
+            git_user="ci",
+            git_email="ci@dummy.com",
+            config={"num": 7, "act": "gelu_fast"},
+            include_optimizer=True,
+            save_traces=False,
```

Comment:
The code after this `push_to_hub_keras` tests for the same thing as `test_push_to_hub`. If the two tests have same code, we are not testing for different behavior. For example, you could test that loading fails or that traces are not there

Reply:
I check if the custom layer exception is raised.